app/src/components/market/profile/view.tsx

```diff
@@ -88,12 +104,19 @@ const ViewWrapper = (props: Props) => {
               }
             />
           )}
+          <TitleValue title="24h Volume" value={lastDayVolume ? `${lastDayVolume} ${collateral.symbol}` : '-'} />
           {showExtraDetails ? (
             <>
-              <TitleValue title={'Mocked Title 1'} value={'Mocked Value 1'} />
-              <TitleValue title={'Mocked Title 2'} value={'Longer Mocked Value to have and idea how it looks'} />
-              <TitleValue title={'Mocked Title 3'} value={'Mocked Value 3'} />
-              <TitleValue title={'Mocked Title 4'} value={'Mocked Value 4'} />
+              <TitleValue
+                title={'Total Pool Tokens'}
+                value={totalPoolShares ? formatBigNumber(totalPoolShares, collateral.decimals) : '0'}
```

Comment:
The pool shares don't necessarily have the same number of decimals as the collateral.